src/components/Navigation.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Link } from 'react-router-dom';
+import './CSSmodules/Navigation.css';
+
+function Navigation() {
+  return (
+    <>
+      <div className="nav-bar">
+        <div className="logo">
+          <h1 className="magicians">Math Magicians</h1>
+        </div>
+        <nav className="nav">
+          <Link className="links" to="/Home">Home</Link>
+          |
+          <Link className="links" to="/calculator">Calculator</Link>
+          |
+          <Link className="links" to="/Quotes">Quotes</Link>
+        </nav>
```

Comment:
- [ ] and the final step to fix the issue mentioned in the main comment is changing the link to the home page to the root path in the navigation menu as well which will look something like below:
`<Link className="links" to="/">Home</Link>`

Reply:
thank you made the required changes